presto-hive/src/test/java/com/facebook/presto/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -4086,6 +4086,22 @@ public void testTemporaryStagingDirectorySessionProperties()
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Test
+    public void testGroupByWithUnion()
+    {
+        assertQuery("SELECT\n" +
+                "      linenumber,\n" +
+                "      'xxx'\n" +
```

Comment:
this test expects an exception on this `xxx` right?

Reply:
Copied this query from the issue. This query would fail without this fix.

Reply:
lol